static/app/views/explore/tables/aggregateColumnEditorModal.tsx

```diff
@@ -67,19 +85,30 @@ export function AggregateColumnEditorModal({
                   options={[]}
                   onColumnChange={c => updateColumnAtIndex(i, c)}
                   onColumnDelete={() => deleteColumnAtIndex(i)}
+                  numberTags={numberTags}
+                  stringTags={stringTags}
+                  groupBys={groupBys}
                 />
               );
             })}
             <RowContainer>
               <ButtonBar gap={1}>
                 <Button
                   size="sm"
-                  aria-label={t('Add a Column')}
-                  onClick={insertColumn}
+                  aria-label={t('Add a Group By')}
+                  onClick={() => insertColumn({groupBy: ''})}
                   icon={<IconAdd isCircled />}
                 >
                   {t('Add a Column')}
                 </Button>
+                <Button
+                  size="sm"
+                  aria-label={t('Add a Visualize')}
+                  onClick={() => insertColumn(new Visualize([DEFAULT_VISUALIZATION]))}
+                  icon={<IconAdd isCircled />}
+                >
+                  {t('Add a Visualize')}
```

Comment:
`Add a Visualize` sounds a little odd. The sidebar uses the `Add a Chart` wording. I wonder if we can have copy that matches both like `Add an Aggregation`. It's a bit more technical sounding but I think users would figure out that an aggregate -> a chart

Reply:
going with `Add an Aggregation` for now and we'll iterate